app/backend/jwt.evaluation.key

```diff
@@ -0,0 +1 @@
+comunismo
```

Comment:
this is the key used to validate the signature?
if so, please move this value to ENV and the variable used to access it to ENV.Sample
this should not be visible, also, change the value